portal-frontend/src/app/features/edit-submission/review-and-submit/submit-confirmation-dialog/submit-confirmation-dialog.component.html

```diff
@@ -0,0 +1,43 @@
+<div mat-dialog-title>
+  <h4>Submit Application</h4>
+</div>
+
+<div mat-dialog-content class="content">
+  <p>
+    Your application will be submitted to the [L/FNG]. After submission, you will be auto-directed to complete the local
+    or first nation government review form in order to submit this application to the Agricultural Land Commission.
+  </p>
+  <div>
+    <div class="subheading2">Terms and Conditions:</div>
+    <mat-checkbox #checkbox1 color="primary" class="checkbox">
+      I/we consent to the use of the information provided in the application and all supporting documents to process the
+      application in accordance with the Agricultural Land Commission Act, the Agricultural Land Reserve General
+      Regulation, and the Agricultural Land Reserve Use Regulation.
+    </mat-checkbox>
+    <mat-checkbox #checkbox2 color="primary" class="checkbox">
+      I/we declare that the information provided in the application and all the supporting documents are, to the best of
+      my/our knowledge, true and correct.
+    </mat-checkbox>
+    <mat-checkbox #checkbox3 color="primary" class="checkbox">
+      I/we understand that the Agricultural Land Commission will take the steps necessary to confirm the accuracy of the
+      information and documents provided. This information will be available for review by any member of the public.
+    </mat-checkbox>
+  </div>
+  <p>
+    If you have any questions about the collection or use of this information, please contact the Agricultural Land
+    Commission.
+  </p>
+  <div>
+    <div class="step-controls">
+      <button mat-stroked-button color="primary" [mat-dialog-close]="false">cancel</button>
+      <button
+        mat-flat-button
+        color="primary"
+        [disabled]="!checkbox1.checked || !checkbox2.checked || !checkbox3.checked"
+        [mat-dialog-close]="true"
+      >
```

Comment:
is there supposed to be `(click)=""` somewhere to call `onSubmit` and `onCancel`?

Reply:
Nope! I have deleted onSubmit and onCancel, [mat-dialog-close]="true" sets the button to close the dialog and passes whatever value in the brackets to any subscribers.